src/release-specification.test.ts

```diff
@@ -657,7 +657,62 @@ ${releaseSpecificationPath}
       });
     });
 
-    it('throws if there are any packages in the release with a major version bump using a literal version, but any of their dependents defined as "peerDependencies" are not listed in the release', async () => {
+    it('throws if there are any packages in the release with a major version bump using the word "major", but any of their dependents defined as "peerDependencies" have no changes since their latest release and are not listed in the release', async () => {
```

Comment:
Nit: I wonder if we should reword this slightly to emphasize that usually a package that has no changes doesn't need to be included in the release, but in this case that doesn't matter:

``` suggestion
    it('throws if there are any packages in the release with a major version bump using the word "major", but any of their dependents defined as "peerDependencies" are not listed in the release, even if they have no changes', async () => {